frontend/ui/widget/menu.lua

```diff
@@ -898,7 +898,12 @@ function Menu:updatePageInfo(select_number)
             self.selected = { x = 1, y = select_number }
         end
         -- update page information
-        self.page_info_text:setText(util.template(_("page %1 of %2"), self.page, self.page_num))
+        -- don't show page x of x if there is only one page
```

Comment:
I like to think of myself as a commentophile, but this one's really quite redundant. :-)
```suggestion
```